nvidia-update.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-set -e
+set -eu
```

Comment:
I think this is a personal preference.

`set -e` is important for runtime correctness - the script makes assumptions (has requirements), if they're not true it's best to bail out.

However, `set -u` is more of a development thing. It helps during development of a script if you specifically don't want to expand undefined bash variables. I don't necessarily want that behaviour.

Is there an error you're hitting (or one you can see) where `set -u` resolves the problem?

Generally speaking, if a change is not addressing an issue it is unnecessary.